soil/templates/soil/partials/dl_status.html

```diff
@@ -1,4 +1,5 @@
 {% load i18n %}
+{% load hq_shared_tags %}
```

Comment:
for the moment I think django-soil technically doesn't depend on HQ. I'm fine removing that abstraction though since from a practical perspective i don't think the library is useful outside of HQ. bonus points if you subtree merge it into the main repo (e.g. ex-submodules)